docs/spec/rpc/swagger.yaml

```diff
@@ -0,0 +1,2680 @@
+                      pub_key:
+                        required:
+                          - "type"
+                          - "value"
+                        properties:
+                          type:
+                            type: "string"
+                            example: "tendermint/PubKeyEd25519"
+                          value:
+                            type: "string"
+                            example: "cOQZvh/h9ZioSeUMZB/1Vy1Xo5x2sjrVjlE/qHnYifM="
+                        type: "object"
+                      voting_power:
+                        type: "string"
+                        example: "8590153"
+                      proposer_priority:
+                        type: "string"
+                        example: "-79515145"
+                    type: "object"
+                type: "object"
+              triggered_timeout_precommit:
+                type: "boolean"
+                example: false
+            type: "object"
+          peers:
+            type: "array"
+            items:
+              type: "object"
+              properties:
+                node_address:
+                  type: "string"
+                  example: "357f6a6c1d27414579a8185060aa8adf9815c43c@68.183.41.207:26656"
+                peer_state:
+                  required:
+                    - "round_state"
+                    - "stats"
+                  properties:
+                    round_state:
+                      required:
+                        - "height"
+                        - "round"
+                        - "step"
+                        - "start_time"
+                        - "proposal"
+                        - "proposal_block_parts_header"
+                        - "proposal_block_parts"
+                        - "proposal_pol_round"
+                        - "proposal_pol"
+                        - "prevotes"
+                        - "precommits"
+                        - "last_commit_round"
+                        - "last_commit"
+                        - "catchup_commit_round"
+                        - "catchup_commit"
+                      properties:
+                        height:
+                          type: "string"
+                          example: "1311801"
+                        round:
+                          type: "string"
+                          example: "0"
+                        step:
+                          type: "number"
+                          example: 3
+                        start_time:
+                          type: "string"
+                          example: "2019-08-05T11:28:49.21730864Z"
+                        proposal:
+                          type: "boolean"
+                          example: false
+                        proposal_block_parts_header:
+                          required:
+                            - "total"
+                            - "hash"
+       
```

Comment:
Is there a way to indicate that this can be either a string or an integer? Right now it actually comes back as an integer. See #3898. I think we're starting to move towards having integers be represented as integers.